src/Common/Commands.ResourceManager.Profile.Test/TenantCmdletTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
I think we need to add some mocked tests for this - we can do it using the existing infrastructure, or using Moq infrastructure (I'm checking some in with the subscription cmdlet)